js/init_and_modules_test.go

```diff
@@ -60,6 +60,7 @@ func (cm *CheckModule) VuCtx(ctx context.Context) {
 }
 
 func TestNewJSRunnerWithCustomModule(t *testing.T) {
+	t.Skip()
```

Comment:
Does this still have to be skipped? If so, let's add a reason for it.

Reply:
IIRC, this was because the test broke when it was ran with `--count 2`, since it tried to register the same module twice. @MStoykov, can we simply register it once with a `sync.Once()`? Or, alternatively, make the test package to be `js` instead of `js_test` and directly mess with the `modules` map in the `InitContext` of the `Bundle`, instead of using `modules.Register()` : https://github.com/k6io/k6/blob/8939092c6c521543eb727202d900ddef07c0511b/js/initcontext.go#L73

Reply:
How about I have it have a random-ish name ? ... the only problem is that with `count=10000` it will probably fill the map ... but :shrug:

Reply:
Eh, :+1: that would work, though I'd prefer you have an `atomic`-ally incremented name instead of a random one, just so we don't have another slightly flaky test that fails once in a blue moon :sweat_smile: